index.js

```diff
@@ -267,26 +290,21 @@ bio: Add 1-2 sentences (or use lorem ipsum)
 
 At the end, this function should return the new array with information added"*/
 
-function addArtist(/* Code here */){
-
-    /* Code here */
-
-  }
-
-/* Task 7: Create a function called lotsOfArt() that takes one argument: 
+function addArtist(array, index, id, name,  genre, years, nationality, bio){
+  array[index].push();
+      return array;
+}
+console.log(artists, 0 , id = 21, name = `Juan Lagunas`, genre = `web design`, years = 2000 - 2020, nationality = `American`, bio = `Lorem ipsum`);
```

Comment:
Your new artist should be created by the addArtist() function instead of console.logging the artist info on its own.